CRM/Civicase/Activity/ContactActivitiesSelector.php

```diff
@@ -55,25 +71,31 @@ public function getActivitiesForContactCount($params) {
    * Returns the original parameters, but without any offsets or limits.
    *
    * @param array $params
+   *   Parameters.
+   *
    * @return array
+   *   Paramaters.
    */
-  private function getParamsWithoutOffsetsAndLimits($params) {
+  private function getParamsWithoutOffsetsAndLimits(array $params) {
     $options = CRM_Utils_Array::value('options', $params, []);
     $options['limit'] = 0;
     $options['offset'] = 0;
     $params['options'] = $options;
+    $params['activity_type_id'] = ['!=' => 'Bulk Email'];
```

Comment:
We might not need to add this actually. I'm assuming that Params is from the API request, so the FE will already have that sent.

Also as the name of the function implies, it's basically to remove offset and limit so it won't be proper to add what is not related.